README.md

```diff
@@ -74,8 +74,8 @@ Before submitting, you must pass all the unit tests and syntax checks by running
 
 ## Notice
 
-* Multiple locales can be built by using comma separated values in the `DEFAULT_LOCALE` variable. i.e: `DEFAULT_LOCALE=en,es,it`.
-* For other 'Serve/Build Options' and 'Test Options', please see `package.json` in details.
+* Multiple locales can be built by using comma-separated values in the `DEFAULT_LOCALE` variable. i.e: `DEFAULT_LOCALE=en,es,it`.
+* For other "Serve/Build Options" and "Test Options", please see `package.json` in details.
```

Comment:
It's not clear to me why these are capitalized and in quotations. I'm also not sure what "in details" means at the end of this sentence. Maybe this?

```suggestion
* For other options, see `package.json`.
```